pkg/clusterinit/onboard/passthrough.go

```diff
@@ -72,21 +74,24 @@ func (s *passthroughGenerator) Generate(ctx context.Context, log *logrus.Entry)
 		for i := range splitStrings {
 			data[i] = []byte(splitStrings[i])
 		}
-
 		pathToManifests[path.Join(clusterRoot, p)] = data
 
 		return nil
 	}); err != nil {
 		return nil, fmt.Errorf("create manifests: %w", err)
 	}
 
+	data := make([]interface{}, 1)
```

Comment:
I guess that we can handle it better than this. Instead of:
```golang
//go:embed ci.openshift.io_multiarchbuildconfigs.yaml
var MultiArchBuildConfigManifest []byte
```
we could declare another `embed.FS` (see my comment above about moving this declaration into its own file):
```golang
//go:embed multiarchbuildconfig/v1/ci.openshift.io_multiarchbuildconfigs.yaml
var MultiArchBuildConfigManifest embed.FS
```
Then we change `passthroughGenerator` such that it would be able to handle multiple `fs.FS`:
```golang
type passthroughGenerator struct {
	manifests []fs.FS // or something similar, this is just to give you an idea
}
```